src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -19,10 +20,27 @@ import { OrganizationFilters } from './OrganizationFilters';
 import { PublisherFilter } from './PublisherFilter';
 import { SeriesFilter } from './SeriesFilter';
 
+const StyledDivider = styled(Divider)(({ theme }) => ({
+  backgroundColor: theme.palette.primary.main,
+}));
+
+const StyledTypography = styled(Typography)({
+  marginBottom: '0.5rem',
+});
+
+const GridRowDivider = () => {
+  return (
+    <Grid item xs={12}>
+      <StyledDivider />
+    </Grid>
+  );
+};
```

Comment:
ah, doh, nå kom jeg på hvorfor vi fikk feilmelding når vi fjerna funksjonene her i stad

```
const GridRowDivider = (
  <Grid item xs={12}>
    <StyledDivider />
  </Grid>
);
```
Må ev. brukes sånn i tsx:
```
<>
  {GridRowDivider}
</>
```
(ikke `<GridRowDivider/>` som blir dobbelt opp)

Går for det samme, så ta den løsningen du selv syns gir mest mening 👍